src/process/shell.cr

```diff
@@ -102,4 +102,60 @@ class Process
   def self.quote_windows(arg : String) : String
     quote_windows({arg})
   end
+
+  # Split a *line* string into the array of tokens in the same way the POSIX shell.
+  #
+  # ```
+  # p Process.split(%q["foo bar" '\hello/' Fizz\ Buzz]) # => ["foo bar", "\\hello/", "Fizz Buzz"]
```

Comment:
```suggestion
  # Process.split(%q["foo bar" '\hello/' Fizz\ Buzz]) # => ["foo bar", "\\hello/", "Fizz Buzz"]
```

We usually don't use `p` in the examples.

Reply:
And it should apply renaming to `Process.parse_arguments`. Thank you.